src/platform/forms/save-in-progress/helpers.js

```diff
@@ -8,6 +9,23 @@ import RoutedSavableReviewPage from './RoutedSavableReviewPage';
 import FormSaved from './FormSaved';
 import SaveInProgressErrorPage from './SaveInProgressErrorPage';
 
+/**
+ * Change the heading level that is rendered inside the saveInProgressIntro.
+ * This is used inside the SaveInProgressIntro component
+ * @param headingLevel {string} - The string of what heading level you want ('h1', 'h2', etc). If you want default h3 pass in null
+ * @param appType {string} - The App Type for your application. This is set in the formConfig file for your form
+ *
+ */
+export function changeSaveInProgressHeadingLevel(headingLevel, appType) {
+  const H = `${headingLevel}`;
+  return (
+    <H className="usa-alert-heading">
+      Save time—and save your work in progress—by signing in before starting
+      your {appType}
+    </H>
+  );
```

Comment:
I would rather this stay in the `SaveInProgressIntro` file so it is immediately obvious what we are getting, rather than importing a function and having to go somewhere else to see what the return value will be.

Reply:
@bkjohnson Yep, I agree, I moved this into the `SaveInProgressIntro` component